barsys/views.py

```diff
@@ -1071,13 +1072,11 @@ def get(self, request, user_id):
 
 
 @api_view(['GET', 'POST'])
-def main_user_purchase_api(request):
-    """
-    List all code snippets, or create a new snippet.
-    """
+def main_purchase_api(request):
     if request.method == 'GET':
-        # serializer = PurchaseSerializer(Purchase.objects.all())
-        return Response('{nice}', status=status.HTTP_200_OK)
+        purchases = Purchase.objects.all()
```

Comment:
Number of purchases should probably be limited, possibly to PybarsysPreferences.Misc.NUM_MAIN_LAST_PURCHASES

Reply:
My idea was, that the client could implement its own filtering if it gets every purchase. But it would probably be more convenient to provide endpoints enabling to query filtered purchases.